actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/util/UserUtil.java

```diff
@@ -643,7 +643,20 @@ public static void validateExternalIds(User user, String operationType) {
       validateUserExternalIds(user);
     }
   }
-
+  public static void validateExternalIds(User user, String operationType, boolean isCustodianOrg) {
+    if (CollectionUtils.isNotEmpty(user.getExternalIds())) {
+      List<Map<String, String>> list = copyAndConvertExternalIdsToLower(user.getExternalIds());
+      user.setExternalIds(list);
+    }
+    //If operation is update and user is custodian org, ignore uniqueness check
+    if (!(JsonKey.UPDATE.equalsIgnoreCase(operationType) && isCustodianOrg)) {
```

Comment:
this must be just isCustodianOrg.
If it is custodian org, then irrespective of operationType - we dont want to check uniqueness.

Reply:
validateExternalIds(User user, String operationType) method is getting use from UPDATE, CREATE and MIGRATE USER. As of now, overloaded method validateExternalIds(User user, String operationType, boolean isCustodianOrg) is getting used only from UPDATE. But  the checkExternalIdUniqueness(user, operationType) call needs operationType. Please confirm.